src/components/InnerComponents/Votes.jsx

```diff
@@ -0,0 +1,47 @@
+import { faCircleDown, faCircleUp } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+const Votes = ({ votes, review_id }) => {
+  const [displayVotes, setDisplayVotes] = useState(votes);
+  const [voteChange, setVoteChange] = useState(0);
+  const [skipInitial, setSkipInitial] = useState(true);
+  const [voteErr, setVoteErr] = useState(null);
+
+  const handleVote = (vote) => {
+    if (vote > -2 && vote < 2) {
```

Comment:
```js
    if (vote > -2 && vote < 2) {
```
The vote is hard coded to be 1 or -1 so would never fail this condition. Not sure if you intended this to limit the amount of votes that can occur but it either should be updated to do so or removed.

Reply:
Was more thinking if someone tried to change it with dev tools